pyfr/libs/conduit.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from ctypes import (POINTER, c_char_p, c_void_p, 
+                    c_double, c_float, c_long, c_longlong, RTLD_GLOBAL)
+from pyfr.ctypesutil import LibWrapper
+
+
+class ConduitWrappers(LibWrapper):
+    _libname = 'conduit'
+    _mode = RTLD_GLOBAL
+
+    # Functions
+    _functions = [
+        (c_void_p, 'conduit_node_append', c_void_p),
```

Comment:
Probably need to override errcheck to check the return values of these methods (assuming NULL means an error).  OpenCL shows how this can be done (although its use case is more complex).